### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by Mihndim2020**
For this pull request I have;
- Implemented a test for a function that creates an list object from corresponding input parameters,
- Implemented  a test to a function that add and deletes an Item from a list in the DOM.

### CODE REVIEW

**Comment by Mihndim2020** [comment]
I am working solo this week. My partner had an emergency.

File: src/andAndDelete.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  test('it creates a list corresponding inputs', () => {
+    createNewTask(taskDescription);
+
+    expect(tasks[tasks.length - 1].description).toBe(taskDescription);
+
+    ul.appendChild(taskList(tasks[tasks.length - 1]));
+
+    expect(ul.innerHTML.includes(taskDescription)).toBe(true);
+  });
```
Comment (mimipeshy) [inline src/andAndDelete.test.js]: - [ ] Let's add some negative test case scenarios as well for example what happens if there is no value in input field

File: src/andAndDelete.test.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  test('it adds and deletes an item from the To Do List in the DOM', () => {
+    const addedTask1 = addTasks(task1.description, task1.completed, task1.index);
+    const addedTask2 = addTasks(task2.description, task2.completed, task2.index);
+
```
Comment (mimipeshy) [inline src/andAndDelete.test.js]: - [ ] Kindly add tests to check if it deletes more than one checked element in the DOM

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Mihndim2020  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 

## Required Changes ♻️
- [x] Create a tests folder and place all your test files there. This will help you organize your code structure
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by Mihndim2020** [comment]
All **required changes** implemented. Optional changes will be implement later due to time constraints.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Mihndim2020 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

yougotit

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**